src/pudl/transform/ferc1.py

```diff
@@ -1962,6 +1962,45 @@ def apply_xbrl_calculation_fixes(
                     },
                 ],
             },
+            "electric_plant_depreciation_changes_ferc1": {
+                "ending_balance": [
+                    {
+                        "calc_component_to_replace": {},
+                        "calc_component_new": {
+                            "name": "starting_balance",
+                            "weight": 1.0,
```

Comment:
How did you determine that these were the calculation components of `ending_balance`? Did you run the calculation and see that it was mostly accurate? Is it based on the components used in other tables? Would be good to document this somewhere.